CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ All notable changes to this project will be documented in this file.
 ## [2.1.4] - 2021-04-06
 
 ### Added
+- chore: Add custom user agent
+- fix: Ensure sdk retry logic is enabled in prod
+- docs: Readme updated
```

Comment:
These are not in 2.1.4: https://github.com/awslabs/service-workbench-on-aws/commits/v2.1.4

Reply:
Corrected.